news/models.py

```diff
@@ -7,3 +7,71 @@ class Subscriber(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Agency(models.Model):
+    user = models.OneToOneField(get_user_model(), on_delete=models.CASCADE)
+    # todo add other fields
+
+    def __str__(self):
+        return self.user.username
+
+
+class Category(models.Model):
+    parent = models.ForeignKey('self', blank=True, null=True, on_delete=models.CASCADE)
+    title = models.CharField(max_length=50, unique=True)
+    description = models.CharField(max_length=250)
+    icon = models.ImageField(blank=True, upload_to='categories')
+    is_enable = models.BooleanField(default=True)
+    created_time = models.DateTimeField(auto_now_add=True)
+    updated_time = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return self.title
+
+
+class News(models.Model):
```

Comment:
Generate a random token for News so that it's GET api be like:
domain/news/{token}
also I think this token shouldn't be longer than 12 characters

Reply:
Done, token added to news model